tools/src/process-allure-reports.py

```diff
@@ -0,0 +1,101 @@
+import os
+import subprocess
+import argparse
+from allure_combine import combine_allure
+
+COMBINE_DIR = 'combine'
+COMPLETE_FILE_NAME = 'index.html'
+PUT_TIMEOUT = 600
```

Comment:
minor: i suggest to mention units for faster understanding
```suggestion
PUT_TIMEOUT = 600 # in seconds
```